File: Java/Odleglosc_w_wyrazach.java
import java.util.Scanner;

class Main
{
    static final String ALPHABET = "ABCDEFGHIJKLMNOPQRSTUVWXYZ";

    public static void main(String[] args) 
    {
        Scanner input = new Scanner(System.in);
        int numberOfWords = input.nextInt();
        input.nextLine();
        for (int i = 0; i < numberOfWords; i++) 
        {
            String word = input.nextLine().toUpperCase();
            System.out.println(letterDifference(word));
        }
        input.close();
    }

    static int letterDifference(String word) 
    {
        int maxIndex = 0;
        int minIndex = ALPHABET.length() - 1;
        for (char letter : word.toCharArray()) 
        {
            int currentIndex = ALPHABET.indexOf(letter);
            if (currentIndex > maxIndex)
                maxIndex = currentIndex;
            if (currentIndex < minIndex)
                minIndex = currentIndex;
        }
        return maxIndex - minIndex;
    }
}

File: Python/Odleglosc_w_wyrazach.py
ALPHABET = "ABCDEFGHIJKLMNOPQRSTUVWXYZ"

def letter_difference(word):
    max_index = 0
    min_index = len(ALPHABET) - 1
    for letter in word:
        current_index = ALPHABET.index(letter)
        if current_index > max_index:
            max_index = current_index
        if current_index < min_index:
            min_index = current_index
    return max_index - min_index

number_of_words = int(input())
for _ in range(number_of_words):
    word = input()
    print(letter_difference(word))